### PR TITLE
DOCS-1938: Update Grafana tutorial R2D2

### PR DESCRIPTION
**Pull request by JessamyT**
One of the images used in the tutorial is also used elsewhere so I updated things accordingly.

### CODE REVIEW

File: docs/tutorials/services/visualize-data-grafana.md [hunk 1]
```diff
@@ -71,14 +71,15 @@ To capture data from your machine and sync to the Viam app, add the data managem
 
 First, add the data management service to your machine to be able capture and sync data:
 
-1. On your machine's **Config** page in the [Viam app](https://app.viam.com), navigate to the **Services** tab.
-1. Click the **Create service** button at the bottom of the page, and select **Data Management**.
-1. Give the service a name, like `viam-data-manager`, then click **Create Service**.
+1. Navigate to your machine's **CONFIGURE** tab in the [Viam app](https://app.viam.com).
+1. Click the **+** button in the left-hand menu and select **Service** from the drop-down create menu.
```
Comment (JessamyT) [inline docs/tutorials/services/visualize-data-grafana.md:75]:
```suggestion
1. Click the **+** button in the left-hand menu and select **Service** from the dropdown.
```

**Review by npentrel — APPROVED** [review]
Lgtm but can you change line 47 to say add the machine and follow the steps on the setup page and link to that? This feels exactly like one of those tutorials where we kind of just say install this without explaining further or saying that you also need to pass the config to connect (since the setup page covers all that I think pointing there makes sense)

**Comment by JessamyT** [comment]
> Lgtm but can you change line 47 to say add the machine and follow the steps on the setup page and link to that? This feels exactly like one of those tutorials where we kind of just say install this without explaining further or saying that you also need to pass the config to connect (since the setup page covers all that I think pointing there makes sense)

I assume you mean line 47 of the ml model tutorial since line 47 of the grafana tutorial isn't about viam-server? The focus of this PR is the Grafana one so I've edited both.

**Comment by npentrel** [comment]
taking over to resolve conflicts and merge

**Pull request merged**